bot/bot.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import os
+from pathlib import Path
+
+import disnake
+from dotenv import load_dotenv
+from disnake.ext import commands
+
+
+logger = logging.getLogger(__name__)
+
+path = Path(__file__)
+parent = path.parents[1]
+load_dotenv(parent.joinpath(".env"))
+DEV_LOG = os.environ.get("DEV_LOG")
```

Comment:
To be explicit, I didn't know what this was by just looking at it.
```suggestion
DEV_LOG_CHANNEL = os.environ.get("DEV_LOG_CHANNEL ")
```